com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeIndexOfIndices.cs

```diff
@@ -57,6 +57,8 @@ internal void Pack(out uint[] vals)
             }
         }
 
+        static uint[] s_PackedValues = new uint[kUintPerEntry];
```

Comment:
While I agree with this change, can we move it inside IndexMetaData ? So that we don't confuse it for something else elsewhere.

Reply:
I'll do that yeah.